docs/cimpl6ToolingReference.md

```diff
@@ -342,83 +343,69 @@ The Content Profile (CP) specifies the profiles to be included in the IG. As exp
 
-    Namespace: obf
-        Patient:
-            BirthDate MS
-        Encounter: NP
-        EpisodeOfCare: NP
+The No-Profile (`NP`) flag is a way to put boundaries around your IG, and eliminate some or all supporting profiles. The No-Profile (`NP`) flag after a class instructs SHR-CLI to _not_ profile a class, and therefore, exclude it from the list of profiles in the IG. To exclude all classes in a namespace, put the `NP` flag after the namespace name.
 
-    Namespace: vital
-    BloodPressure:
-        Components.ObservationComponent MS
-        RelevantTime MS
-    BodyHeight:
-        DataValue MS
-        RelevantTime MS
-    BodyWeight:
-        DataValue MS
-        RelevantTime MS
-        ...
-```
+The `NP` flag can be applied to both `Entries` and `Groups`. An example of applying `NP` to a `Group`, if there is a model of Address that is different than FHIR's Address data type, then specifying `Address: NP` will use FHIR's Address rather than a profiled version of Address.
+
+Because the web of dependencies can be hard to predict, a good way to determine which classes should be designated with an `NP` flag is to generate the IG, then go to the `out/fhir/profiles` directory to see the complete list of profiles generated. If any profiles in that directory are extraneous to the purpose of the IG, go back to the CP and apply the `NP` flag to the corresponding class. However,the `NP` flag should not be applied to any class that is a parent class of one of the classes you want in your IG. This may interfere with mapping rules inherited from that parent class.
+
+***
+**NOTE:** The "No Profile" feature is available in SHR-CLI 6.6.0 and higher
+***
 
 #### Specifying MustSupport Elements
 
-In FHIR, [MustSupport](https://www.hl7.org/fhir/conformance-rules.html#mustSupport) is a boolean flag which allows a profile to indicate that an implementation must be able to process that element in a FHIR instance if it exists.
+In FHIR, [MustSupport](https://www.hl7.org/fhir/conformance-rules.html#mustSupport) is a boolean flag which allows a profile to indicate that an implementation must be able to process that element in a FHIR instance if it exists. In the CP file, MustSupport elements are designated with the `MS` flag after the property name.
 
-MustSupport elements are designated with the `MS` tag after the property name.
+MustSupport elements are not included in the data model because different use cases may require support for different elements. For example, one use case may require a patient's date of death and another might not, but the data model for date of death should be the same wherever it is used.
```

Comment:
Potential confusion with reference to MustSupport elements - with the flag after the property name.
Code block Property (on line 376), bold element on 376 and 378 (confirm the reference to elements on 378 is to the element in a FHIR profile).

Reply:
ok